lib/sql.go

```diff
@@ -3,79 +3,59 @@ package lib
 import (
 	"bytes"
 	"context"
-	"encoding/json"
-	"fmt"
 
 	"github.com/qri-io/qri/sql"
 )
 
-// SQLMethods encapsulates business logic for the qri search command
-// TODO (b5): switch to using an Instance instead of separate fields
+// SQLMethods groups together methods for SQL
 type SQLMethods struct {
-	inst *Instance
+	d dispatcher
 }
 
-// NewSQLMethods creates SQLMethods from a qri Instance
-func NewSQLMethods(inst *Instance) *SQLMethods {
-	return &SQLMethods{inst: inst}
-}
+// Name returns the name of this method group
+func (m SQLMethods) Name() string { return "sql" }
 
-// CoreRequestsName implements the requests interface
-func (m SQLMethods) CoreRequestsName() string { return "sql" }
+// Attributes defines attributes for each method
+func (m SQLMethods) Attributes() map[string]AttributeSet {
+	return map[string]AttributeSet{
+		"exec": {AESQL, "POST"},
+	}
+}
 
-// SQLQueryParams defines paremeters for the exec Method
-// ExecParams provides parameters to the execute command
+// SQLQueryParams defines paremeters for running a SQL query
 type SQLQueryParams struct {
-	Query        string
-	OutputFormat string
-	ResolverMode string
+	Query  string
+	Format string
```

Comment:
Not a massive fan of the  `OutputFormat`  -> `Format` switch. To me the test in `lib/integration_test.go` now reads kinda like the format of the _query_ is JSON.

That said, this might be in the name of aligning field names. Care to clarify why you made the change?

Reply:
Yes, done to align the field name with other commands. For example `list` and `diff` both use `Format` to talk out about their output format, and all three of these fall into the "aggregation" category of commands. It also matches the command-line flag `--format` already in use for the command.

I sort of see what you mean about the ambiguity of input/output format, but the context makes it clearer.